### PR TITLE
Updated research.rst

### PR DESCRIPTION
**Pull request by anranjiao**

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -1,9 +1,11 @@
+
```
Comment (lululxvi) [inline docs/user/research.rst:1]: Why adding this empty line?

File: docs/user/research.rst [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 Research
 ========
 
 DeepXDE has been used in
 
-- > 80 universities, e.g.,
+- > 150 universities, e.g.,
+  `Harvard University <https://www.harvard.edu/>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:8]: Delete `/` at the end of each url.

File: docs/user/research.rst [hunk 3]
```diff
@@ -105,45 +174,138 @@ DeepXDE has been used in
...
   `BirenTech Research <https://www.birentech.com>`_,
   `ExxonMobil <https://corporate.exxonmobil.com>`_,
   `General Motors <https://www.gm.com>`_,
   `RocketML <https://www.rocketml.net>`_,
-  `Saudi Aramco <https://www.aramco.com>`_
+  `Saudi Aramco <https://www.aramco.com>`_,
+  `Shell India Markets Private Limited <https://www.shell.in/about-us/contact-us.html>`_,
+  `SoftServe <https://www.softserveinc.com/en-us>`_,
+  `Fraunhofer Institute for Integrated Systems and Device Technology IISB <https://www.iisb.fraunhofer.de/>`_,
+  `Oak Ridge National Laboratory <https://www.ornl.gov/>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:201]: Oak Ridge is lab, not industry.

File: docs/user/research.rst [hunk 4]
```diff
@@ -105,45 +173,138 @@ DeepXDE has been used in
   `Athinoula A. Martinos Center for Biomedical Imaging <https://www.martinos.org>`_,
   `Friedrich-Alexander-Universität Erlangen-Nürnberg Research Center for Mathematics of Data <https://mod.fau.eu>`_,
   `Data Observatory Foundation <https://dataobservatory.net>`_,
-  `Fraunhofer Institute for Integrated Systems and Device Technology IISB <https://www.iisb.fraunhofer.de>`_
+  `Fraunhofer Institute for Integrated Systems and Device Technology IISB <https://www.iisb.fraunhofer.de>`_,
+  `ONERA, The French Aerospace Lab <https://www.onera.fr/en>`_,
+  `Ministry of Natural Resources (China) <https://www.mnr.gov.cn>`_,
+  `Zhejiang Lab <https://en.zhejianglab.com>`_,
+  `Los Alamos National Laboratory <https://www.lanl.gov>`_,
```
Comment (lululxvi) [inline docs/user/research.rst:180]: Move Los Alamos and Oak Eidge after Argonne

File: docs/user/research.rst [hunk 5]
```diff
@@ -105,45 +175,136 @@ DeepXDE has been used in
...
+  `Zhejiang Lab <https://en.zhejianglab.com>`_,
+  `United Kingdom Atomic Energy Authority <https://www.gov.uk/government/organisations/uk-atomic-energy-authority>`_,
+  `Russian Academy of Sciences <https://new.ras.ru/en>`_,
+  `Chinese Academy of Sciences <https://english.cas.cn>`_,
+  `AppliedAI Institute for Europe <https://www.appliedai.de>`_,
+  `Rutherford Appleton Laboratory <https://www.ukri.org/who-we-are/stfc/locations/rutherford-appleton-laboratory>`_,
+  `Mathematical Sciences Research Laboratory <http://math.uth.gr/laboratory/?lang=en>`_,
+  `German Aerospace Center <https://www.dlr.de/en>`_,
+  `Materials Center Leoben Forschung GmbH <https://www.mcl.at>`_
 - industry, e.g.,
```
Comment (lululxvi) [inline docs/user/research.rst:190]:
```
- > 10 industry, e.g.,
```

File: docs/user/research.rst [hunk 6]
```diff
@@ -105,45 +175,136 @@ DeepXDE has been used in
...
+#. F\. Fonseca. `A solution of a 3D cartesian poisson-boltzmann equation <https://www.m-hikari.com/ces/ces2023/ces1-2023/p/fonsecaCES1-2023.pdf>`_. *Contemporary Engineering Sciences*, 16(1):1-10, 2023.
+#. L\. Fritschi and K. Lenk. `Parameter inference for an astrocyte model using machine learning approaches <https://www.biorxiv.org/content/10.1101/2023.05.16.540982v1.abstract>`_. *bioRxiv*, p. 2023-05, 2023.
+#. Z\. Gong, Y. Chu, and S. Yang. `Physics-informed neural networks for solving 2-D magnetostatic fields <https://ieeexplore.ieee.org/abstract/document/10141630?casa_token=1fiPdhDF_ewAAAAA:RH2pE-IZj1prPA4IMx6Dn3fLLkTu9jkp09g-CKPC7WbW6xN9iwT-pYJNAJcXp0d4eOj8f0ILmg>`_. *IEEE Transactions on Magnetics*, 59(11):1-5, 2023.
+#. M\. A. Haddou. `Quasi-normal modes of near-extremal black holes in dRGT massive gravity using physics-informed neural networks (PINNs) <https://arxiv.org/abs/2303.02395>`_. 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:270]: Just to confirm, these papers use DeepXDE, not only cite DeepXDE, right?

File: docs/user/research.rst [hunk 7]
```diff
@@ -105,45 +175,136 @@ DeepXDE has been used in
...
 #. L\. Sliwinski, & G. Rigas. `Mean flow reconstruction of unsteady flows using physics-informed neural networks <https://www.cambridge.org/core/journals/data-centric-engineering/article/mean-flow-reconstruction-of-unsteady-flows-using-physicsinformed-neural-networks/FA2A09B976B0ACE4C8C2CEA9205C540D>`_. *Data-Centric Engineering*, 4, p.e4, 2023.
 #. E\. Lorin, & X. Yang. `Schwarz waveform relaxation-learning for advection-diffusion-reaction equations <https://www.sciencedirect.com/science/article/pii/S0021999122007203?casa_token=hZsXqzdjoXoAAAAA:czw_GhUSII7WfE0UehrS6UXadNdldXqO_pN-zRU4r480Wn_fB_PswseUR_fnd4WmPPYk-Tx2YVU>`_. *Journal of Computational Physics*, 473, p.111657, 2023.
-#. F\. Fonseca. `A solution of a 3d Cartesian Poisson-Boltzmann equation <http://www.m-hikari.com/ces/ces2023/ces1-2023/p/fonsecaCES1-2023.pdf>`_. *Contemporary Engineering Sciences*, Vol. 16, no. 1, 1-10, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:138]: How these papers are ordered?

File: docs/user/research.rst (hunk 6 above)
Comment (anranjiao) [reply docs/user/research.rst:270]: Yes

File: docs/user/research.rst (hunk 7 above)
Comment (anranjiao) [reply docs/user/research.rst:138]: They are sorted by time based on google scholar

**Pull request merged**